modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/form.jsp

```diff
@@ -70,6 +121,16 @@ portletDisplay.setURLBack(backURL);
 			);
 		}
 
+		function <portlet:namespace />getCategoryIds(categories) {
```

Comment:
@seltong Are you using this function anywhere?

Reply:
I'm using only in one place. I will remove this function and create a new one just like you suggested in the comment below.